webtool/gradle/common/src/commonMain/kotlin/Pilot.kt

```diff
@@ -7,4 +7,78 @@ import kotlin.js.JsExport
 @JsExport
 data class Pilot(
     val name: String,
+    val numHouseholds: int,
+    val households: List<Households>,
+    val numCompanies: int,
+    val companies: List<Company>,
+    val numSolarfarms: int,
+    val solarfarms: List<Solarfarm>,
+    val numWindfarms: int,
+    val windfarms: List<Windfarm>,
+    val numBatteries: int,
+    val batteries: List<Battery>,
+    val numHeatStorages: int,
+    val heatStorages: List<HeatStorage>,
 )
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Households(
+    val type: String,
+    val numHouseholds: int,
+    val hasPVRatio: double,
+    val hasHeatPumpRatio: double,
+    val hasChargePointRatio: double,
+    val hasHomeBatteryRatio: double,
+)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Company(
+    val name: String,
+)
+
+/*
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Utility(
+    val name: String,
+    val type: String,
+)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class ConsumptionAsset(
+    val type: String,
+    val consumption_kW: double,
+)
+*/
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Solarfarm(
+    val production_kW: double,
+)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Windfarm(
+    val production_kW: double,
+)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Battery(
+    val capacity_kWh: double,
+    val peakPower_kW: double,
+)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class HeatStorage(
+    val capacity_kWh: double,
```

Comment:
Dit kunnen we afleiden uit de andere eigenschappen. Daarom moeten we dit niet opslaan.

Je zou evt (later) een functie kunnen toevoegen

```kotlin
data class HeatStorage(
    val water_m3: Double,
    val minTemp: Double,
    val maxTemp: Double,
) {
    fun getCapacity_kWh(): Double {
        val specificHeatCapacity = 4.18 // kJ/kg/K

        return water_m3 * 1000 * specificHeatCapacity * (maxTemp - minTemp) / 3600
    }
}
```

Reply:
waterVolume_m3
minTemp_degC
maxTemp_degC